src-v2/NTS.Domain.Setup/Entities/Phase.cs

```diff
@@ -4,19 +4,19 @@
 
 public class Phase : DomainEntity, ISummarizable, IImportable
 {
-    public static Phase Create(Lap lap, int recovery, int rest) => new(lap, recovery, rest);
+    public static Phase Create(Loop Loop, int recovery, int rest) => new(Loop, recovery, rest);
 
-    public static Phase Update(int id, Lap lap, int recovery, int rest) => new(id, lap, recovery, rest);
+    public static Phase Update(int id, Loop Loop, int recovery, int rest) => new(id, Loop, recovery, rest);
 
     [JsonConstructor]
-    public Phase(int id, Lap selectedLap, int recovery, int rest)
+    public Phase(int id, Loop selectedLoop, int recovery, int rest)
```

Comment:
Isn't this `selectedLoop`  name a problem? Does `selected` bring meaningful context?